ProcessQueue/__init__.py

```diff
@@ -215,19 +233,20 @@ def main(msg: func.QueueMessage):
         # Create a dictionary with field names and field ids 
         # as key value pairs
         # e.g ("Field Name":<Field ID>)
-        namexids=dict(zip(field_names,field_ids))
-
+        namexids=dict(zip(field_ids,field_names))

```

Comment:
Has the order of this changed because that's what `get_field_id()` expects? If so, could you rename this to `id_x_names` to better reflect its structure? Thanks.